mne/coreg.py

```diff
@@ -77,9 +77,8 @@ def create_default_subject(mne_root=None, fs_home=None, update=False,
 
     Parameters
     ----------
-    mne_root : None | str
-        The mne root directory (only needed if MNE_ROOT is not specified as
-        environment variable).
+    mne_root : None
+        This argument is not used anymore.
```

Comment:
Ah okay. And it does only have 3 arguments, so making people use `kwargs` for one release cycle isn't the end of the world. Feel free to deprecate if you want, no strong feeling from my end.